packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```
@@ -14,12 +14,12 @@
   "pci_database_add_step5_title": "Configurez vos options",
   "pci_database_add_name": "Nommez votre service de base de données",
   "pci_database_add_network_mode_label": "Type de réseau",
-  "pci_database_add_network_mode_public": "Public",
-  "pci_database_add_network_mode_private": "Privé",
+  "pci_database_add_network_mode_public": "Réseau public (Internet) ",
+  "pci_database_add_network_mode_private": "Réseau privé (vRack)",
   "pci_database_add_network_mode_public_incompatible": "L'offre sélectionnée n'est pas compatible avec les réseaux publics.",
   "pci_database_add_network_mode_private_incompatible": "L'offre sélectionnée n'est pas compatible avec les réseaux privés.",
   "pci_database_add_privateNetwork_label": "Réseau privé attaché",
-  "pci_database_add_privateNetwork_description": "Par défaut, les bases de données possèdent un réseau public. Vous pouvez également ajouter un réseau privé, mais vous perdrez la configuration que vous venez d’établir.",
+  "pci_database_add_privateNetwork_description": "Seuls les services (serveurs dédiés, Vms, clusters K8s, …) connectés a ce <b>réseau privé (vRack)</b> pourront communiquer avec votre base de données. ",
```

avoid to use the HTML template into content:
```suggestion
  "pci_database_add_privateNetwork_description": "Seuls les services (serveurs dédiés, Vms, clusters K8s, …) connectés a ce {{htmlBoldTag}}réseau privé (vRack){{htmlBoldTagEnd}} pourront communiquer avec votre base de données. ",
```

and use them into a dedicated HTML template like so (sample):
`data-translate-values="{ htmlBoldTag: '<b>', htmlBoldTagEnd: '</b>' }"`

---

Done here: https://github.com/ovh/manager/pull/7460/commits/e248d3b57bd62ecfa6b51212c378a16c2683497a